src/pycontw2016/urls.py

```diff
@@ -33,6 +33,7 @@
 urlpatterns += [
     url(r'^ccip/', include('ccip.urls')),
     url(r'^api/sponsors/', include('sponsors.api.urls')),
+    url(r'^api/events/', include('events.api.urls', namespace="events")),
```

Comment:
according to api document the url should be like "api/conference/talk/{id}/"

Reply:
URL != API path, it means you can find these data in 2020's website with URL.
So I think api/events/ is fine.